pom.xml

```diff
@@ -886,189 +929,6 @@
                     </dependency>
                 </dependencies>
             </plugin>
-            <plugin>
```

Comment:
Question for discussion: I've moved forbiddenapis to a profile, so it can be disabled manually during development builds. Happy to back this change out again, if someone feels that's a bad idea.